<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>692</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
background-color: 	rgb(50,50,50);
}</string>
   </property>
   <widget class="QPushButton" name="build_model">
    <property name="geometry">
     <rect>
      <x>839</x>
      <y>631</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
	background-color: 	rgb(32,178,70);
	color:white;
	border-radius:2px;
	font: 200 13pt Apple;
}

QPushButton::pressed {
	background-color: 	rgb(27, 152, 60);
}</string>
    </property>
    <property name="text">
     <string>Build</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_model">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>631</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
	background-color: 	rgb(120,120,120);
	color:white;
	font: 200 13pt Apple;
	border-radius:2px;
}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="move_label">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>407</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotate_label">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>489</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QLabel" name="scale_label">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>568</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_transform">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>380</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 600 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Transform</string>
    </property>
   </widget>
   <widget class="QLabel" name="move_label_x">
    <property name="geometry">
     <rect>
      <x>729</x>
      <y>407</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="move_label_y">
    <property name="geometry">
     <rect>
      <x>729</x>
      <y>432</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="move_label_z">
    <property name="geometry">
     <rect>
      <x>729</x>
      <y>455</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotate_label_z">
    <property name="geometry">
     <rect>
      <x>729</x>
      <y>537</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotate_label_y">
    <property name="geometry">
     <rect>
      <x>729</x>
      <y>514</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotate_label_x">
    <property name="geometry">
     <rect>
      <x>729</x>
      <y>489</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="mode_label">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>601</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_filename">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>41</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
background-color: none;
}</string>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_ver">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>68</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Vertexes</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_f">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>97</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 200 13pt Apple;
	color: white;
background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Facets</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>598</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	font: 200 13pt Apple;
	background-color: 	rgb(30,30,30);
	color:white;
	padding:5px;
	border-radius:2px;
}

QComboBox::down-arrow {
background-color: transparent;
 border-left: 5px solid none;
border-right: 5px solid none; 
border-top: 5px solid #808080; 
}


</string>
    </property>
    <item>
     <property name="text">
      <string>parallel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>central</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="file_name_label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>36</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: 	rgb(90,90,90);
border-radius: 2px;
padding-right:4px;
color:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_v">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>64</y>
      <width>289</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: 	rgb(90,90,90);
border-radius: 2px;
padding-right:4px;
color:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_f">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>92</y>
      <width>289</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: 	rgb(90,90,90);
border-radius: 2px;
padding-right:4px;
color:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_choose_file">
    <property name="geometry">
     <rect>
      <x>838</x>
      <y>130</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
	background-color: 	rgb(32,178,70);
	color:white;
	border-radius:2px;
	font: 200 13pt Apple;
}

QPushButton::pressed {
	background-color: 	rgb(27, 152, 60);
}</string>
    </property>
    <property name="text">
     <string>Choose file...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_transform_2">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 600 13pt Apple;
	color: white;
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string>Information</string>
    </property>
   </widget>
   <widget class="Viewer" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>651</width>
      <height>635</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Viewer {
	background-color: 	rgb(0,50,50);
}</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>566</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	font: 200 13pt Apple;
    border-style: solid;
	background-color: 	rgb(90,90,90);
	color:white;
	padding:4px;
border-radius:2px;
}</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_rotate_z">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>533</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	font: 200 13pt Apple;
    border-style: solid;
	background-color: 	rgb(90,90,90);
	color:white;
	padding:4px;
border-radius:2px;
}</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-5.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_rotate_y">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>509</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	font: 200 13pt Apple;
    border-style: solid;
	background-color: 	rgb(90,90,90);
	color:white;
	padding:4px;
border-radius:2px;
}</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-5.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_move_z">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>452</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	font: 200 13pt Apple;
    border-style: solid;
	background-color: 	rgb(90,90,90);
	color:white;
	padding:4px;
border-radius:2px;
}</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_rotate_x">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>485</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	font: 200 13pt Apple;
    border-style: solid;
	background-color: 	rgb(90,90,90);
	color:white;
	padding:4px;
border-radius:2px;
}</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-5.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_move_x">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>404</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	font: 200 13pt Apple;
    border-style: solid;
	background-color: 	rgb(90,90,90);
	color:white;
	padding:4px;
border-radius:2px;
}</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_move_y">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>428</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	font: 200 13pt Apple;
    border-style: solid;
	background-color: 	rgb(90,90,90);
	color:white;
	padding:4px;
border-radius:2px;
}</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="file_name_label_2">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>36</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: 	rgb(90,90,90);
border-radius: 2px;
padding-right:4px;
color:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>file_name_label_2</zorder>
   <zorder>label_f</zorder>
   <zorder>label_v</zorder>
   <zorder>file_name_label</zorder>
   <zorder>build_model</zorder>
   <zorder>reset_model</zorder>
   <zorder>move_label</zorder>
   <zorder>rotate_label</zorder>
   <zorder>scale_label</zorder>
   <zorder>label_transform</zorder>
   <zorder>move_label_x</zorder>
   <zorder>move_label_y</zorder>
   <zorder>move_label_z</zorder>
   <zorder>rotate_label_z</zorder>
   <zorder>rotate_label_y</zorder>
   <zorder>rotate_label_x</zorder>
   <zorder>mode_label</zorder>
   <zorder>info_filename</zorder>
   <zorder>info_ver</zorder>
   <zorder>info_f</zorder>
   <zorder>comboBox</zorder>
   <zorder>button_choose_file</zorder>
   <zorder>label_transform_2</zorder>
   <zorder>widget</zorder>
   <zorder>doubleSpinBox</zorder>
   <zorder>spin_rotate_z</zorder>
   <zorder>spin_rotate_y</zorder>
   <zorder>spin_move_z</zorder>
   <zorder>spin_rotate_x</zorder>
   <zorder>spin_move_x</zorder>
   <zorder>spin_move_y</zorder>
   <zorder>horizontalSlider</zorder>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
		background-color: 	rgb(30,30,30);
	color: white;
}

QMenuBar::hovered{
background-color:green;
}</string>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menu_settings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_settings"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header location="global">viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
